src/current/v23.2/column-level-encryption.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Column Level Encryption
+summary: SQL users are required to pass encryption keys when reading from or writing to specific columns
+toc: true
+docs_area: reference.sql
+---
+
+CockroachDB v23.2 and later support _column level encryption_. Using this feature, you can encrypt one or more of the columns in each row of a database table.
+
+Column level encryption can be useful for compliance scenarios such as adhering to PCI or GDPR.
+
+Once a column is encrypted, SQL users are required to pass an encryption key when reading from or writing to that column. This prevents users without the encryption key from taking actions like the following:
+
+- Reading sensitive information (like _personally identifiable information_ ([PII](https://en.wikipedia.org/wiki/Personal_data))) when accessing the tables.
+- Adding or updating records without encrypting the specified columns.
+
+{{site.data.alerts.callout_info}}
+The "column-level encryption" feature described on this page specifically refers to either symmetric-key encryption or public-key encryption of data, **not** hashing functions operating on data.
+{{site.data.alerts.end}}
+
+{% include enterprise-feature.md %}
+
+## Available Functions
+
+### `encrypt` and `encrypt_iv`
+
+The `encrypt` and `encrypt_iv` functions encrypts a column's data with a given key and cipher method. For more information, see [Cryptographic functions]({% link {{ page.version.version }}/functions-and-operators.md %}#cryptographic-functions).
+
+For usage examples, see:
+
+- [Encrypt using the `encrypt` function](#encrypt-using-the-encrypt-function).
+- [Encrypt using the `encrypt_iv` function](#encrypt-using-the-encrypt_iv-function).
+
+These functions require an [enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}). If you do not have an enterprise license, you will see an error message like the following if you try to use them:
+
+~~~
+ERROR:  encrypt(): use of pgcrypto cipher functions requires an enterprise license. see https://cockroachlabs.com/pricing for details on how to enable enterprise features
```

Comment:
We've updated the error message in https://github.com/cockroachdb/cockroach/pull/111865, it says the following now:
```
decrypt(): use of this cryptographic function (https://www.cockroachlabs.com/docs/stable/functions-and-operators#cryptographic-functions) requires an enterprise license. see https://cockroachlabs.com/pricing for details on how to enable enterprise features
```

Reply:
thanks, I've changed this and the other instance of the error message to use this text